kafka-ui-api/src/test/java/com/provectus/kafka/ui/service/BrokerServiceTest.java

```diff
@@ -47,13 +45,18 @@ void getBrokersNominal() {
 
   @Test
   void getBrokersNull() {
-    assertThatThrownBy(() -> brokerService.getBrokers(null)).isInstanceOf(NullPointerException.class);
```

Comment:
I removed this tests in https://github.com/provectus/kafka-ui/pull/1652, because they are not informative, IMHO. We should not have scenarios that we pass non-existing clusters to service layer- it should be validated before.